mux/craigslist.rb

```diff
@@ -47,19 +52,34 @@ def self.create_listing_from_result(result, survey)
     date = DateTime.strptime r['PostedDate'].to_s, '%s'
 
     # Creating a listing
-    l = Listing.new location: location,
-                ask: r.fetch('Ask') { :default } ,
-                bedrooms: r['Bedrooms'],
-                title: r['PostingTitle'],
-                posting_date: date,
-                survey: survey,
-                source: @@source,
-                payload: r.to_json,
-                uid: r['PostingURL']
+    l = Listing.find_or_initialize_by(uid: r['PostingURL'])
+
+    # Track which fields are changing so that we have a sense of hwo to logically do deduplication
+    fields_changed = []
+    unless l.new_record?
```

Comment:
Oh awesome!! I think that would actually be a much better plan and would allow us to log more stuff. I'll open an issue for it at some point down the road..